<?xml version="1.0" encoding="UTF-8"?>

<launch>
        <arg name="v" default="0.7"/>
        <arg name="w" default="0"/>
        <arg name="d" default="7"/>
        <arg name="min_x" default="0"/>  
        <arg name="max_x" default="9"/>
        <arg name="min_y" default="0"/>
        <arg name="max_y" default="9"/>
    <node pkg="turtlesim" type="turtlesim_node" name="turtlesim_node"/>
    <node pkg="turtlesim_tutorial" type="move_inside_room.py" name="move_turtle" output="screen" >
        <param name="v" value="$(arg v)"/>
        <param name="w" value="$(arg w)"/>
        <param name="d" value="$(arg d)"/>
        <param name="min_x" value="$(arg min_x)"/>
        <param name="max_x" value="$(arg max_x)"/>
        <param name="min_y" value="$(arg min_y)"/>
        <param name="max_y" value="$(arg max_y)"/>
    </node>
</launch>